### ISSUE TITLE
Web UI: jobs table does not have helpful failure messages

### ISSUE THREAD

**Issue by nstewart** [cockroachdb/cockroach#20990]
**FEATURE REQUEST**
I ran a restore job that failed:

![image](https://user-images.githubusercontent.com/894257/34263759-904211f4-e63e-11e7-9c76-3a347df6ff54.png)

There is no way as a user to figure out what happened from this view as far as I can tell. Fortunately, I still had my terminal open so I could see an error message, but this should really have made it to the UI.

**Comment by a-robinson** [comment]
You can also get it from the `system.jobs` table, but I definitely agree with this request.

**Comment by dianasaur323** [comment]
Ah, this is a good point. I think the admin UI team could take this on in the next release. Perhaps for now, we should add a link on Failed to some docs where you can see the error message in systems.job. cc @vilterp @couchand

**Comment by couchand** [comment]
Maybe a tooltip @vilterp?

**Comment by vilterp** [comment]
Yeah, or a column that displays a short error name and you click to expand to a full description… (Not sure what the actual errors look like)

**Comment by dianasaur323** [comment]
@josueeee any thoughts on what this might look like?
